src/main.html

```diff
@@ -84,6 +84,7 @@
     <script type="text/javascript" src="./js/utils/VtxDeviceStatus/TrampDeviceStatus.js"></script>
     <script type="text/javascript" src="./js/utils/VtxDeviceStatus/SmartAudioDeviceStatus.js"></script>
     <script type="text/javascript" src="./js/utils/VtxDeviceStatus/Rtc6705DeviceStatus.js"></script>
+    <script type="text/javascript" src="./js/helper.js"></script>
```

Comment:
Wouldn't need this if https://github.com/betaflight/betaflight-configurator/blob/c8d0490b2ffa66615ea37da0e9a7282af8d01077/src/js/gui.js#L77 is used.